publish-github-release/index.js

```diff
@@ -38,8 +40,10 @@ async function setupCredentials(user, pass)
 
 async function run()
 {
+    const modulesJsonUrl = "https://raw.githubusercontent.com/VirtoCommerce/vc-modules/master/modules_v3.json";
```

Comment:
modulesJsonUrl should be defined as an input parameter for the action
with default value "https://raw.githubusercontent.com/VirtoCommerce/vc-modules/master/modules_v3.json"

Reply:
Fixed